hack/build.sh

```diff
@@ -2,18 +2,24 @@
 
 set -e -x -u
 
-# makes builds reproducible
```

Comment:
i dont think we need to change ./hack/build.sh as its used for development, and development version should probably stay as "develop" (similar to https://github.com/vmware-tanzu/carvel-imgpkg/blob/develop/hack/build.sh)